samples/src/source/tasks/ContentTask.bs

```diff
@@ -0,0 +1,29 @@
+import "pkg:/source/roku_modules/log/LogMixin.brs"
+import "pkg:/source/roku_modules/maestro/core/Utils.brs"
+import "pkg:/source/roku_modules/maestro/core/Request.brs"
+import "pkg:/source/roku_modules/maestro/core/MIOC.brs"
+
+@task("ContentTask", "Task")
+@strict
+class ContentTask
+  private log = new log.Logger("ContentTask")
+
+  @inject("styleManager")
+  private styleManager
+
+  ' @injectClass("mc.net.Request")
+  private request = new mc.net.Request()
+
+  private config = mc.getPath(m.styleManager@.getStyle("app"), "server", {})
+
+  private function execute(args as mc.types.assocarray)
+    command = lcase(mc.getPath(args, "command", ""))
```

Comment:
I appreciate you making this example; but can we make it more useful for other people. Would you please actually load some data and parse it - nothing special, get some items from some json dump somewhere and create a few nodes to show in a rowlist.. - just grabbing the title will be enough.
like: https://jsonplaceholder.typicode.com/photos

Reply:
> I appreciate you making this example; but can we make it more useful for other people. Would you please actually load some data and parse it - nothing special, get some items from some json dump somewhere and create a few nodes to show in a rowlist.. - just grabbing the title will be enough.
> like: https://jsonplaceholder.typicode.com/photos

I will do it!